integration/docker/dockerfile-common.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+function installLibfuse {
+  git clone https://github.com/cheyang/libfuse.git
```

Comment:
use  alluxio/libfuse?

Reply:
oh i didn't even know this existed. would prefer this change too assuming it is a valid substitution

Reply:
@ssz1997 change to alluxio/libfuse  fuse_2_9_5_customize_multi_threads branch?

Reply:
@LuQQiu The problem with the branch with logging is that, if the job is very heavy, the user log would be flooded with the line "libfuse: max_idle_thread = 64", which won't be very helpful.